go.mod

```diff
@@ -42,3 +42,5 @@ require (
 	golang.org/x/sys v0.0.0-20220128215802-99c3d69c2c27 // indirect
 	golang.org/x/term v0.0.0-20210927222741-03fcf44c2211
 )
+
+replace github.com/containers/storage => github.com/containers/storage v1.38.3-0.20220301151551-d06b0f81c0aa
```

Comment:
The dependency above needs to be updated, without using `replace`; `replace` is ignored when importing c/image into other users.

Reply:
thanks for catching that, @mtrmac !